pkg/proto/runtime/v1/dapr_tracing_test.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runtime
+
+import (
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/exp/slices"
+)
+
+// Tests that all requests messages include the "AppendSpanAttribute" method.
+func TestRequestsIncludeAppendSpanAttribute(t *testing.T) {
```

Comment:
The purpose of this test is to make sure that all messages defined in the proto whose name ends in "Request" include the `AppendSpanAttribute`. Unit tests will fail if that's not included.